doc/EFFECTS_JSON.md

```diff
@@ -481,7 +501,7 @@ As it moves up through the different intensities, its name will change. Its desc
 changes, with no additional description added.
 
 As it moves up through the intensity levels its effects will be:
-```C++
+```JSON
```

Comment:
Some of these code blocks (the stuff between \`\`\`) is neither C++ nor JSON, this here for example.

In that case, leave out the language tag, the block will be formated with syntax highlighting.